### ISSUE TITLE
Wiki shows wrong pinout (RGB and PWR-PROBE flipped)

### ISSUE THREAD

**Issue by devdrik** [FYSETC/FYSETC-Cheetah-v2#17]
The wiki for the cheetah 2 shows a wrong pinout:
In [this wiki](https://wiki.fysetc.com/Cheetah_Board_V20/#43-pin-definition) the [pinout image](https://wiki.fysetc.com/images/Cheetah_V2.0_pinout.jpg) has the RGB and PWR-PROBE pins switched. This lead to me killing my LEDs. The GitHub readme already has a correct version of the image [this one](https://github.com/FYSETC/FYSETC-Cheetah-v2/blob/main/images/Cheetah_V2.0_pinout.jpg)

Please update the wiki!

**Comment by ashthespy** [comment]
@devdrik Did you get manage to get this working? I ran into the same issue, and thankfully didn't manage to shot out my LEDs. 
LEDs work fine on other MCUs, but I couldn't get it working with klipper. Any hints?

**Comment by devdrik** [comment]
@ashthespy Yes, when I used the correct pins ([this picture shows the correct LED pins](https://github.com/FYSETC/FYSETC-Cheetah-v2/blob/main/images/Cheetah_V2.0_pinout.jpg)) everything worked fine.
